addons/gamification/models/res_users.py

```diff
@@ -16,6 +16,28 @@ class Users(models.Model):
     rank_id = fields.Many2one('gamification.karma.rank', 'Rank')
     next_rank_id = fields.Many2one('gamification.karma.rank', 'Next Rank')
 
+    @api.depends('karma_tracking_ids.new_value')
+    def _compute_karma(self):
+        self.env['gamification.karma.tracking'].flush_model()
```

Comment:
Will that be triggered by the consolidation?

Seems dangerous, as it will most likely consolidate a lot of stuff and run a big query for a lot of users to update ... well nothing as consolidate will not impact current karma ^^
(Unless the last karma change is old? Value will stay the same anyway).

Maybe a context_key or something?
Or maybe using my "insert into select" trick will solve this?
To double-check.